styles.css

```diff
@@ -784,6 +781,16 @@ ul {
   height: 24px;
 }
 
+.btn-link-source {
+  background-color: #f55800;
+  text-decoration: none;
+  color: white;
+  padding: 10px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
```

Comment:
- [ ] Great jon on working on the Pop up design you have one small thing to fix just decrease the gap between this two buttons 🔥💯😊

- Your design
![image](https://user-images.githubusercontent.com/58553711/188889117-b4bf3a26-110d-4b8b-9bfe-714a32e0ea7b.png)

- Figma design 
![image](https://user-images.githubusercontent.com/58553711/188889192-4c86f2f4-94c0-4dd7-acd9-97703a8895d3.png)

Reply:
ok i use in this section the gap, thank you very much